templates/components/api-server/template/server/middleware.json

```diff
@@ -10,6 +10,26 @@
         "credentials": true,
         "maxAge": 86400
       }
+    },
+    "helmet#xssFilter": {
+      "params": {
+        "setOnOldIE": true
```

Comment:
> On old versions of Internet Explorer, this creates a vulnerability (see here and here), and so the header is set to 0 to disable it. To force the header on all versions of IE, add the option:

I believe we should not enable this option by default, as it would create a security vulnerability.